scripts/ripe-atlas

```diff
@@ -1,38 +1,75 @@
 #!/usr/bin/env python
 
-import importlib
 import os
 import re
 import sys
 
-from ripe.atlas.tools.commands.base import Command
+from ripe.atlas.tools.commands.base import Command, Factory
 from ripe.atlas.tools.exceptions import RipeAtlasToolsException
+from ripe.atlas.tools.helpers.colours import colourise
 
 
 class RipeAtlas(object):
 
+    DEPRECATED_ALIASES = {
+        "measurement": "measurement-info",
+        "measurements": "measurement-search",
+        "probe": "probe-info",
+        "probes": "probe-search",
+    }
+
     def __init__(self):
         self.command = None
         self.args = []
         self.kwargs = {}
 
+    def _show_usage(self):
+        usage = "Usage: ripe-atlas <command> [arguments]\n\n"
+        usage += "Commands:\n"
+        longest_command = 0
+        classes = []
+        for c in Command.get_available_commands():
+            if c == "shibboleet":
+                continue
+            cmd_class = Command.load_command_class(c)
+            classes.append(cmd_class)
+            cmd_name = cmd_class.get_name()
+            if len(cmd_name) > longest_command:
+                longest_command = len(cmd_name)
+        for cmd_cls in classes:
+            usage += "\t{} {}\n".format(
+                cmd_cls.get_name().ljust(longest_command + 1),
+                cmd_cls.DESCRIPTION,
+            )
+        usage += (
+            "\nFor help on a particular command, try "
+            "ripe-atlas <command> --help"
+        )
+        raise RipeAtlasToolsException(usage)
+
```

Comment:
Just a minor thing, it feels a bit weird to have a function name show something and raising at the end. When I was looking at the code below that is calling this function I thought the program flow would continue after showing usage. Would it be better if you return the usage string and raise it under the if?